src/Nethermind/Nethermind.Evm/TransactionProcessing/GasConsumed.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Nethermind.Evm.TransactionProcessing;
 
-public readonly record struct GasConsumed(long SpentGas, long OperationGas)
+public readonly record struct GasConsumed(long SpentGas, long OperationGas, object? PolicyData = null)
```

Comment:
I would love to remove `PolicyData` from here if possible
But probably would have to make a lot of things generic and might not be worth it